### PR TITLE
🚧 Allow to disable SSL

### PR DESCRIPTION
**Pull request by bakayolo**
- no need to deploy cert-manager
- it will not create any certificates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,43 +21,10 @@ The chart will do the following:
...
-
-and you should get something similar to
-
-```sh
-NAME                                       READY   STATUS    RESTARTS   AGE
-cert-manager-766d5c494b-wl4bq              1/1     Running   0          33s
-cert-manager-cainjector-6649bbb695-l5rb2   1/1     Running   0          33s
-cert-manager-webhook-68d464c8b-hvpf6       1/1     Running   0          33s
-```
+- Kubernetes cluster with at least a node having 4GB or more memory.
```
Comment (sumitkolhe) [inline README.md]: typo: 
- at least a node -> at least one node
- having 4GB or more memory -> having 4GB memory or more.

File: README.md [hunk 2]
```diff
@@ -81,20 +50,48 @@ helm upgrade --install argo \
               --namespace argo argo/argo
 ```
 
-Check argo is running fine.
+Check that argo is running fine.
```
Comment (sumitkolhe) [inline README.md]: typo -
- check that -> check if

File: README.md [hunk 3]
```diff
@@ -81,20 +50,48 @@ helm upgrade --install argo \
...
+argo-server-7869fd4b96-xn8gw               1/1     Running   0          62s
+argo-workflow-controller-b68ffccb5-jx7vq   1/1     Running   0          62s
 ```
 
-and you should get something similar to
+#### If SSL is enabled
+
+- [Cert Manager](https://cert-manager.io/docs/)
+Notes: KintoHub has been tested with cert-manager v0.15.0.
+- You must have a domain name ready to be used. KintoHub only support Cloudflare at the moment, you can create a free account and transfer your domain ownership easily. Please create an issue if you want to add more providers.
```
Comment (sumitkolhe) [inline README.md]: typo - 
- only support -> only supports

File: README.md [hunk 4]
```diff
@@ -81,20 +50,48 @@ helm upgrade --install argo \
...
 ```sh
+kubectl create namespace cert-manager
+helm repo add jetstack https://charts.jetstack.io
+helm upgrade --install cert-manager \
+              --version v0.15.0 \
+              --set installCRDs=true \
+              --namespace cert-manager jetstack/cert-manager
+```
+
+Check cert-manager is running fine.
```
Comment (sumitkolhe) [inline README.md]: typo - 
- check cert-manager -> check if cert-manager

**Review by sumitkolhe — CHANGES_REQUESTED** [review]
Just minor typo changes

**Comment by bakayolo** [comment]
@sumitkolhe all your comments have been fixed, please review again. Thanks!

**Review by sumitkolhe — APPROVED** [review]
LGTM

**Pull request merged**